packages/aws-lambda-graphql/src/createDynamoDBEventProcessor.ts

```diff
@@ -35,7 +35,7 @@ class PubSub {
     return createAsyncIterator(
       this.events
         .filter(event => names.includes(event.event))
-        .map(event => event.payload),
+        .map(event => JSON.parse(event.payload)),
```

Comment:
could you please add a check if `event.payload` is actual `string` for backward compatibility?